docs/configuration-reference/platforms/aws.md

```diff
@@ -203,6 +203,7 @@ worker_pool "my-worker-pool" {
 | `asset_dir`                   | Location where Lokomotive stores cluster assets.                                                                                                                                                                                                                                                             | -               | string       | true     |
 | `cluster_name`                | Name of the cluster. **NOTE**: It must be unique per DNS Zone and region.                                                                                                                                                                                                                                    | -               | string       | true     |
 | `tags`                        | Optional details to tag on AWS resources.                                                                                                                                                                                                                                                                    | -               | map(string)  | false    |
+| `os_arch`                     | Flatcar Container Linux architecture to use (x86\_64, arm64).                                                                                                                                                                                                                                                | "x86\_64"        | string       | false    |
```

Comment:
Actually, maybe we can use https://registry.terraform.io/providers/hashicorp/aws/latest/docs/data-sources/ec2_instance_type#supported_architectures to detect the architecture automagically using Terraform, based on instance type?